src/passes.jl

```diff
@@ -306,6 +306,7 @@ end
 function process!(::SetupBuildDirectory, page::Page, content, block)
     dir = dirname(page.build)
     isdir(dir) || mkpath(dir)
+    addstaticfiles!(dir)
```

Comment:
The `false` tells the calling method, https://github.com/MichaelHatherly/Lapidary.jl/blob/7cb5b2ee8b3fd23b91c47b2acbc7dee471355e61/src/passes.jl#L31, whether or not to try the next action on the block or to move on to the next block. I think I might change the `Bool` to a custom `Enum` to make that move clear and also add some additional choices of what to do after an action.

Once we start trying to build larger docs, such as the Julia manual, I suspect that we'll need to optimise the number of times some things get run. For now though just checking the page's parent is `build/` should be sufficient.